models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-goodstore/free-shipping.spec.ts

```diff
@@ -0,0 +1,20 @@
+import type {
+  TCartDiscountDraft,
+  TCartDiscountDraftGraphql,
+} from '../../../types';
+import freeShipping from './free-shipping';
+
+describe('with the preset `freeShipping`', () => {
+  it('should return a cart discount draft', () => {
+    const cartDiscountDraft = freeShipping().build<TCartDiscountDraft>();
+
+    expect(cartDiscountDraft).toMatchInlineSnapshot();
```

Comment:
You need to run this test once so that Jest (test running tool) fills in the data here for the inline snapshot. Otherwise I would expect it to fail.